### ISSUE TITLE
FollowMe NPE

### ISSUE THREAD

**Issue by squilter** [DroidPlanner/DroneKit-Android#84]
```
java.lang.NullpointerException
at
org.droidplanner.core.gcs.roi.ROIEstimator.updateRIO(ROIEstimator.java:90)
at
org.droidplanner.core.gcs.roi.ROIEstimator$1.run(ROIEstimattor.java:30)
at
...
```
using version 1.2.9

**Comment by ne0fhyk** [comment]
@squilter is the error easily repeatable?

**Comment by squilter** [comment]
No. Not sure how it happened.
 

 



 

 

-----Original Message-----

**Comment by ne0fhyk** [comment]
Fixed by PR #99
